File: Evented/src/main/java/name/benjaminabbitt/evented/opentracing/TracerSupport.java
package name.benjaminabbitt.evented.opentracing;

import io.jaegertracing.Configuration;
import io.opentracing.Tracer;

public class TracerSupport {
    public static Tracer getTracer() {
        return Configuration.fromEnv("java-test").withSampler(new Configuration.SamplerConfiguration().withType("const").withParam(1)).getTracer();
    }
}

File: Evented/src/main/kotlin/name/benjaminabbitt/evented/opentracing/TracerSupport.kt
package name.benjaminabbitt.evented.opentracing

import io.jaegertracing.Configuration
import io.opentracing.Tracer

object TracerSupport {
    @JvmStatic
    val tracer: Tracer
        get() = Configuration.fromEnv("java-test").withSampler(Configuration.SamplerConfiguration().withType("const").withParam(1)).tracer
}